<launch>
    <!-- Start Service Server For move_bb8_in_circle Service -->
    <node pkg="my_custom_srv_msg_pkg" type="bb8_move_custom_service_server.py" name="service_move_bb8_in_circle_custom_server"  output="screen">
    </node>
</launch>